policyengine_us/parameters/gov/states/la/tax/credits/child_care_expense_credit/rate/refundable.yaml

```diff
@@ -0,0 +1,36 @@
+description: Louisiana provides this percentage of the child care expenses a refundable credit.
```

Comment:
Louisiana matches this percentage of the eligible expense based on the quality rating of the child care facility.